datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad

```diff
@@ -134,45 +134,45 @@
-- offshore_suitable_for_wind = 258.769864280363
+- number_of_buildings = 1693.0
+- number_of_cars = 37955.0
+- number_of_residences = 42035.0
+- number_of_inhabitants = 90837.0
+- offshore_suitable_for_wind = 30060.0
 - residences_roof_surface_available_for_pv = 0.711628922860919
 - buildings_roof_surface_available_for_pv = 0.728028587209362
-- energetic_emissions_other_ghg_industry = 0.0
-- energetic_emissions_other_ghg_energy = 0.0
-- energetic_emissions_other_ghg_transport = 0.0072470399
-- energetic_emissions_other_ghg_buildings = 0.002791977394
-- energetic_emissions_other_ghg_households = 0.0
-- energetic_emissions_other_ghg_agriculture = 0.0
-- non_energetic_emissions_co2_chemical_industry = 0.0
-- non_energetic_emissions_co2_other_industry = 0.0
-- non_energetic_emissions_co2_agriculture_manure = 0.0
-- non_energetic_emissions_co2_agriculture_soil_cultivation = 0.0
+- energetic_emissions_other_ghg_industry = 8.39e-05
+- energetic_emissions_other_ghg_energy = 0.001133317
+- energetic_emissions_other_ghg_transport = 0.012291215
+- energetic_emissions_other_ghg_buildings = 0.000356292
+- energetic_emissions_other_ghg_households = 0.001808202
+- energetic_emissions_other_ghg_agriculture = 0.003701698
+- non_energetic_emissions_co2_chemical_industry = 7.31e-05
+- non_energetic_emissions_co2_other_industry = 0.002391933
+- non_energetic_emissions_co2_agriculture_manure = 4.42e-05
+- non_energetic_emissions_co2_agriculture_soil_cultivation = 2.35e-05
 - non_energetic_emissions_co2_waste_management = 0.0
 - non_energetic_emissions_other_ghg_chemical_industry = 0.0
-- non_energetic_emissions_other_ghg_other_industry = 0.00377017695566
-- non_energetic_emissions_other_ghg_agriculture_fermentation = 0.0
-- non_energetic_emissions_other_ghg_agriculture_manure = 0.0
-- non_energetic_emissions_other_ghg_agriculture_soil_cultivation = 0.0
-- non_energetic_emissions_other_ghg_agriculture_other = 0.013517608
-- non_energetic_emissions_other_ghg_waste_management = 0.0
-- indirect_emissions_co2 = 0.0
-- number_of_detached_houses = 1264.6966216862857
-- number_of_apartments = 21719.207699301372
-- number_of_semi_detached_houses = 890.849004689444
-- number_of_corner_houses = 4920.782132261459
-- number_of_terraced_houses = 14249.464542061442
-- base_dataset = nl
+- non_energetic_emissions_other_ghg_other_industry = 0.003070046
+- non_energetic_emissions_other_ghg_agriculture_fermentation = 0.004647302
+- non_energetic_emissions_other_ghg_agriculture_manure = 0.001632877
+- non_energetic_emissions_other_ghg_agriculture_soil_cultivation = 0.004647203
+- non_energetic_emissions_other_ghg_agriculture_other = 0.0
+- non_energetic_emissions_other_ghg_waste_management = 0.001570105
+- indirect_emissions_co2 = 0.002223454
+- number_of_detached_houses = 1235.0220116757584
+- number_of_apartments = 21209.59218585511
+- number_of_semi_detached_houses = 869.9462867260025
+- number_of_corner_houses = 4805.321801129293
+- number_of_terraced_houses = 13915.11771461384
```

Comment:
One more sanity check: There are less homes in 2019 than there were in 2016.

Reply:
Again it seems right with the 2019 data we have from CBS, could again be a mistake in the previous dataset